lib/network/modules/components/edges/util/EdgeBase.js

```diff
@@ -110,7 +110,9 @@ class EdgeBase {
   _drawDashedLine(ctx, values, viaNode, fromPoint, toPoint) {  // eslint-disable-line no-unused-vars
     ctx.lineCap = 'round';
     let pattern = [5,5];
-    if (Array.isArray(values.dashes) === true) {
```

Comment:
This comparison is ONLY truthy from direct edge object property values.  The inherited values from the `options.edges` property, somehow get converted into an array-like object, that Array.isArray does not recognize, and as shown below, when using `JSON.stringify` to inspect values.dashes it spits out an empty object.

<img width="195" alt="screen shot 2018-05-21 at 12 40 37 pm" src="https://user-images.githubusercontent.com/37982292/40323961-4204471e-5cf4-11e8-9cb1-0c6c869d53a5.png">